ArcGIS-toolbox/ASF_Tools.pyt

```diff
@@ -415,21 +431,16 @@ class ReclassifyRTC(object):
 
         # Check licensing
         self.isLicensed()
-        
+
         # Define parameters
         inRTC = parameters[0].valueAsText
         rc_outpath = parameters[1].valueAsText
         rc_outname = parameters[2].valueAsText
-        thresh = parameters[3].value
         outYN = parameters[4].valueAsText
 
-        threshstr = str(thresh)
-
         # Run the code to reclassify the image
-        
-        rcname = str(rc_outpath+'\\'+rc_outname)
-        values = "-1000.000000 %s 1;%s 1000.000000 NODATA" % (threshstr, threshstr)
-        reclass = arcpy.gp.Reclassify_sa(inRTC, "VALUE", values, rcname, "DATA")
```

Comment:
Per Heidi `Reclassify_sa` does needed work, even if the returned `reclass` value isn't used

Reply:
@hjkristenson Is it okay to just make the funciton call without assignment here?